firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -78,4 +74,22 @@ private static String encodeFidBase64UrlSafe(byte[] rawValue) {
             Charset.defaultCharset())
         .substring(0, FID_LENGTH);
   }
+
+  private static byte[] getBytesFromUUID(UUID uuid) {
```

Comment:
This method name is not correct (you are doing heaps of FID specific stuff in here)
I recommend that you create a method that actually does nothing but get the bytes from a UUID and move all the FID specific stuff into the createRandomFid method.